### ISSUE TITLE
0.99.0 libgit2.pc file gets not installed in correct place

### ISSUE THREAD

**Issue by saxa** [libgit2/libgit2#5472]
#5412 
# Reproduction steps
mkdir builddir; cd builddir
cmake -DCMAKE_INSTALL_PREFIX=/usr \    
        -DCMAKE_C_FLAGS=$PKG_CFLAGS \                                                                                   
        -DCMAKE_CXX_FLAGS=$PKG_CXX_FLAGS \                                                                              
        -DTHREADSAFE=ON \
        -DLIB_INSTALL_DIR=/usr/lib64 \
        -DCMAKE_BUILD_TYPE=Release ..
make 
make install DESTDIR=/tmp/pack-libgit2

### Expected behavior
To get the libgit2.pc file installed in the correct prefixed dir on x86_64
like /usr/lib64/pkgconfig/libgit2.pc

### Actual behavior
The libgit2.pc file gets installed in a /usr/lib/pkgconfig instead of /usr/lib64/pkgconfig
directory.
Other libs get installed in the correct /usr/lib64/ dir.

### Version of libgit2 (release number or SHA1)
0.99.0

### Operating system(s) tested
Slackware64 Linux current

**Comment by pks-t** [comment]
Thanks a lot for your report! I'm a bit confused by the fact that you already include #5412, which fixes the exact issue you're reporting. Are you saying that you found an additional bug that wasn't yet fixed by #5412?

**Comment by saxa** [comment]
Hi Patrick, no. I included it because github suggested it to me, before I reported I tried the search for libgit2.pc but just the LTO bug appeared, so I filled the bug and then when clicked oncommit that 
bug showed up. So I just put it as a refference, mine seems is a duplicate. 
Sorry for the noise, I saw the fix, I have not tried to build master to see if it fixes it on mine setup but I think it should.

Thanks and I think we can close this one or mark it as duplicate. I dont know how you over here work :) .
Enjoy.
